services/common/src/components/forms/SteppedForm.tsx

```diff
@@ -104,12 +133,29 @@ const SteppedForm = (props) => {
   return (
     <Row>
       <Col span={6} className="stepped-form-menu-container">
-        <Menu className="stepped-form" mode="inline" selectedKeys={tabs[tabIndex]} items={items} />
+        <Menu
+          className="stepped-form"
+          mode="inline"
+          selectedKeys={[tabs[tabIndex].toString()]}
+          items={items}
+        />
       </Col>
       <Col span={18}>
         {children && (
           <div className="stepped-form-form-container">
-            <Form layout="vertical">{children.find((child) => child.key === tabs[tabIndex])}</Form>
+            <FormWrapper
+              name={formName}
+              onSubmit={() => {}}
+              initialValues={initialValues}
+              isEditMode={isEditMode}
+              reduxFormConfig={{
+                touchOnBlur: true,
+                touchOnChange: false,
```

Comment:
I'm wondering if it might be worth passing this value as a prop with a default of false.  I can see some potential scenarios where we would want touchOnChange to be true (even though we don't need it now)

Reply:
I'm thinking that when we get to the point of wanting variable configuration, to have it take in FormWrapperProps to allow you to override _everything_. But at this point it feels like over-engineering.

Reply:
I decided to do it.